File: classes/parte-3-valores-tipos-y-variables/045-numeros-reales-punto-flotante-precision-y-decimales/implementaciones/c/main.c
#include <stdio.h>

int main(void) {
    double a, b;
    if (scanf("%lf %lf", &a, &b) != 2) return 1;
    printf("suma=%.2f producto=%.2f\n", a + b, a * b);
    return 0;
}

File: classes/parte-3-valores-tipos-y-variables/045-numeros-reales-punto-flotante-precision-y-decimales/implementaciones/go/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	line, _ := bufio.NewReader(os.Stdin).ReadString('\n')
	f := strings.Fields(line)
	a, _ := strconv.ParseFloat(f[0], 64)
	b, _ := strconv.ParseFloat(f[1], 64)
	fmt.Printf("suma=%.2f producto=%.2f\n", a+b, a*b)
}
